vasttools/query.py

```diff
@@ -1893,17 +1919,16 @@ def _add_files(self, row: pd.Series) -> Tuple[str, str, str]:
             rms_dir,
             rms_file_fmt
         ))
-        
+
         if not image_file.is_file():
             conv_image_file = Path(str(image_file).replace('.restored',
                                                            '.restored.conv')
-                                                           )
+                                   )
             if conv_image_file.is_file():
                 image_file = conv_image_file
-                rms_file = Path(str(image_file).replace('.restored',
-                                                        '.restored.conv')
-                                                        )
-
+                rms_file = Path(str(rms_file).replace('.restored',
+                                                      '.restored.conv')
+                                )
```

Comment:
```suggestion
                rms_file = rms_file.with_suffix('.restored.conv')
```
Is `rms_file` already a Path here? If so you could do the suggestion.

Reply:
The problem with that is that the `.restored.conv` is in the middle of the file name, not the suffix

Reply:
And as far as I can tell, there's no nice way to do a simple replace on a Path

Reply:
Ah of course!